warehouse/static/sass/blocks/_faq-group.scss

```diff
@@ -52,4 +52,8 @@
       margin-bottom: 15px;
     }
   }
+
+  em {
+    font-style: normal
```

Comment:
I think we'll still want some emphasis here, perhaps this should make the text bold instead?

Reply:
👍